### ISSUE TITLE
Babel plugin prevents access to var external to class where class method contains `super`

### ISSUE THREAD

**Issue by overlookmotel** [overlookmotel/livepack#248]
### Problem

Input:

```js
class C {}
let D = class extends C {
  foo() {
    return super.foo, D;
  }
};

const K = D;
D = 1;
const k = new K();
console.log( typeof k.foo() );
```

When original code is run, it logs "number". With Livepack's Babel transform enabled, it logs "function".

### Cause

Reason is that the Babel plugin names the class `D` where it was unnamed before, so `D` always refers to the class, rather than the external var `D`.

It is doing this as part of mechanism to capture `super`, but it's malfunctioning in this case.

Extract from Babel-transformed code:

```js
let D = class D extends C {
  foo() {
    livepack_tracker(
      /*livepack_track:{"id":5,"scopes":[{"blockId":1,"varNames":["D"],"blockName":"index"},{"blockId":4,"varNames":["D"],"constNames":["D"],"blockName":"D"}],"isStrict":true,"isProtoMethod":true,"superVarName":"D"}*/
      () => [[livepack_scopeId_1, D], [livepack_scopeId_1, D]]
    );
    return super.foo, D;
  }
};
```

This also causes the class to be serialized incorrectly as one `D` in tracker comment refers to wrong var.

### Solution

The class should not be named unless var definition is `const`. Or maybe Babel plugin should not name classes at all.

Related to #247.

**Comment by overlookmotel** [comment]
The class-naming is intended to (a) reduce use of temp vars in Babel-ized output and (b) remove need for a scope for the class constructor's `super` call.

Suggestion in https://github.com/overlookmotel/livepack/issues/48#issuecomment-887731490 would remove advantage (b).

If Babel plugin didn't name classes, this would remove the problem. It's probably not worth it just for reduced Babel-ized output.

**Comment by overlookmotel** [comment]
Fixed in 2e4401f.
